awx/main/models/jobs.py

```diff
@@ -159,6 +159,12 @@ class Meta:
             "If enabled, Tower will act as an Ansible Fact Cache Plugin; persisting "
             "facts at the end of a playbook run to the database and caching facts for use by Ansible."),
     )
+    gather_event_types = models.CharField(
```

Comment:
The next thing people are going to ask for is an `/api/v2/settings/jobs/` toggle that lets you set the default behavior globally.

Reply:
I guess the question is... how visible do we want to make this? A while ago I would have said... lets not show this at all. Now I'm not so sure. It might be worth having this even be exposed through the UI.

Reply:
why is this per-template instead of global?

Reply:
I think, at least at first, this *needs* to be per template... having it global would affect smaller jobs that need more detail.

I feel like a global default could be set later.